### PR TITLE
Store Mocks

### PR DESCRIPTION
**Pull request by k4ylZ137**

### CODE REVIEW

File: tests/store_test.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+from unittest.mock import patch
+from io import StringIO
+from store import store 
+
+class TestStore(unittest.TestCase):
+    
+    @patch('store.database_manager')
+    @patch('sys.stdout', new_callable=StringIO)
+    
+    def test_init_with_no_pokemon(self, mock_stdout, mock_db_manager):
```
Comment (liammcmanu) [inline tests/store_test.py:11]: Just leaving a wee not here, this doesn't run, should just be a simple change to:

```    def test_init_with_no_pokemon(self, mock_stdout, mock_db_manager):
        mock_db_manager.return_value.get_user.return_value = [('username','Johnny', 'Black', 100)]
        store('username', [])
        self.assertIn( 'Johnny, Your Pokebits 100 PokeBts 🪙', mock_stdout.getvalue())
        self.assertIn('What would you like to buy today?', mock_stdout.getvalue())
        self.assertIn('Oh there is no Pokemon to display!', mock_stdout.getvalue())
```

File: tests/store_test.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        mock_db_manager.return_value.get_user.return_value = [('username','Johnny', 'Black', 100)]
+        store - store('username', [])
+        expected_output - "Oh there is no Pokemon to display!\n"
+        self.assertEqual(mock_stdout.getvalue(), expected_output)
+
+    def test_init_with_pokemon(self, mock_stdout, mock_db_manager):
```
Comment (liammcmanu) [inline tests/store_test.py:17]: This on the hand, a bit iffy, not sure if you ran it but there is gonna be user input which will leave the test in an infinite loop, I think, I'm sure there's a way around it, but worse comes to worse we can just chop it, we've already got this quite well covered from the first test with changes

**Review by liammcmanu — COMMENTED** [review]
Honestly not bad, make the changes and should be golden

**Comment by Arran22** [comment]
Code has ben updated to suit comments, tests should pass now!

**Pull request merged**